src/loaders/expressLoader.ts

```diff
@@ -0,0 +1,29 @@
+import { MicroframeworkSettings } from "microframework";
+import express from "express";
+
+import config from "../config";
+import apiV1 from "../api";
+
+
+export const expressLoader = (settings: MicroframeworkSettings) => {
+    const app = express();
+
+    app.use(express.json());
+    app.use(config.api.v1.prefix, apiV1());
+
+    app.all("/status", (req, res) => {
+        res.end("ok");
+    });
+
+    app.all("*", (req, res) => {
+        res.status(404).end();
+    })
+
+    app.listen(config.expressPort, () => {
+        console.log(`Server listening on port: ${config.expressPort}`);
+    });
+    app.on("error", () => {
+        process.exit(1);
```

Comment:
it's not really good practice to exit the app if there is error

Reply:
having errors it's expected behavior and we should send the error to the client. We should stop app only if the erorr is really critical

Reply:
Updated

Reply:
let's remove `process.exit` since it's not critical if we have error

Reply:
Done